Gopkg.toml

```diff
@@ -67,34 +67,41 @@
   source = "https://github.com/simonferquel/yaml"
   revision = "c86e64ed9581b7588e736f0c3e6ecc02cc22996e"
 
+# Use GitHub mirror to not require Mercurial
+[[override]]
+  name = "bitbucket.org/ww/goautoneg"
```

Comment:
Oh; let me know if this looks ok to you; this dependency hasn't been touched in 4 years, so I don't expect any thing to change in there.

Fetching it from this github mirror at least made it work (perhaps we should fork it into the docker org though?)

Reply:
+1 with forking it